### PR TITLE
Friendlier MURs missing fields

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
“Unavailable” /
Is better than “Unknown” /
Make templates use it.

https://github.com/18F/openFEC-web-app/issues/1651

Marking as “do not merge” because I don't have tests yet, but I wanted to add the PR so that @jenniferthibault could have a look at it.

A good MUR to look at for it is 3952.

### CODE REVIEW

**Comment by jenniferthibault** [comment]
👍 

This is wonderful /
Here are screenshots for others /
They may be useful

screen shot 2016-11-02 at 5 30 35 pm

screen shot 2016-11-02 at 5 31 00 pm

screen shot 2016-11-02 at 5 30 50 pm

File: openfecwebapp/templates/legal-archived-mur.html [hunk 1]
```diff
@@ -35,18 +35,19 @@ <h2 class="t-ruled--bold t-ruled--bottom">Summary</h2>
...
         </div>
         <div class="legal-mur__property">
-        <strong>CLOSE DATE:</strong> {{ mur.close_date | date('%Y-%m-%d') }}
+        <strong>CLOSE DATE:</strong> {{ legal.mur_date(mur.close_date) }}
         </div>
         <div class="legal-mur__property">
         <strong>SUBJECT:</strong>
-        {% for node in mur.subject %}
+        {{ legal.mur_subjects(mur.subject) }}
+        {#% for node in mur.subject %}
```
Comment (vrajmohan) [inline openfecwebapp/templates/legal-archived-mur.html]: Shouldn't we remove the commented out code?

File: openfecwebapp/templates/macros/legal.html [hunk 2]
```diff
@@ -75,3 +75,68 @@ <h2 class="message__title">No results</h2>
...
+
+{% macro mur_date(date_property) %}
+{% if date_property %}
+    {{ date_property | date('%Y-%m-%d') }}
+{% else %}
+    Unavailable
+{% endif %}
+{% endmacro %}
+
+{% macro mur_participants(mur) %}
```
Comment (vrajmohan) [inline openfecwebapp/templates/macros/legal.html]: Did we forget to use this macro in the template?

File: openfecwebapp/templates/macros/legal.html [hunk 3]
```diff
@@ -75,3 +75,68 @@ <h2 class="message__title">No results</h2>
...
+      <h2 class="t-ruled--bold t-ruled--bottom u-no-margin">Participants</h2>
+      <table class="data-table data-table--text data-table--heading-borders">
+        <thead>
+          <tr>
+            <th>Relationship</th>
+            <th>Name</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% if 'complainants' in mur and mur.complainants %}
```
Comment (vrajmohan) [inline openfecwebapp/templates/macros/legal.html]: Do we need the 2nd clause? Isn't it covered by the `for` below?

File: openfecwebapp/templates/macros/legal.html [hunk 4]
```diff
@@ -75,3 +75,68 @@ <h2 class="message__title">No results</h2>
...
+            {% for complainant in mur.complainants %}
+            <tr class="row-color-normal">
+              {% if loop.first %}
+              <td rowspan="{{ mur.complainants | length }}">Complainant</td>
+              {% endif %}
+              <td>{{ complainant }}</td>
+            </tr>
+            {% endfor %}
+          {% endif %}
+          {% if 'respondents' in mur and mur.respondents %}
```
Comment (vrajmohan) [inline openfecwebapp/templates/macros/legal.html]: Do we need the 2nd clause? Isn't it covered by the `for` below?

Comment (tadhg-ohiggins) [inline openfecwebapp/templates/macros/legal.html]: Good catch.

File: openfecwebapp/templates/legal-archived-mur.html (hunk 1 above)
Comment (tadhg-ohiggins) [inline openfecwebapp/templates/legal-archived-mur.html]: Sorry, missed a commit that removes these.

File: openfecwebapp/templates/macros/legal.html (hunk 2 above)
Comment (tadhg-ohiggins) [inline openfecwebapp/templates/macros/legal.html]: Sorry, missed a commit that removes these.

**Comment by tadhg-ohiggins** [comment]
@jenniferthibault I merged your recent paragraph changes into this, could you check to see that it looks the way it's supposed to?

**Comment by jenniferthibault** [comment]
Ah will do now, sorry for causing the merge conflicts!

**Comment by jenniferthibault** [comment]
@tadhg-ohiggins almost there, looks like one thing got tangled up: the "Unavailable" text for the SUBJECT field should be in-line, like the others, but it seems to be breaking onto another. Once that's fixed, 👍 

screen shot 2016-11-03 at 3 04 03 pm

**Pull request merged**